src/helpers/utility.js

```diff
@@ -96,10 +96,15 @@ export function gasToQtum(gas) {
   return new BigNumber(gas).multipliedBy(gasCostBN).toNumber();
 }
 
-export function getEndTimeCountDownString(unixSeconds, locale, localeMessages) {
+/**
+ * Converts a duration to the countdown display string.
+ * @param unixDiff {Number} The duration to convert. Formatted in unix time format.
+ * @param locale {Object}} Locale object that the Intl is using.
+ * @param localeMessages {Object} LocalMessages object that the Intl is using.
```

Comment:
nit: the type should be before the param name:
`@param {number} unixDiff Description goes here`